packages/bpk-component-badge/src/BpkBadge.js

```diff
@@ -31,6 +33,7 @@ export const BADGE_TYPES = {
   outline: 'outline',
 };
 
+// TODO Does this need to change?
```

Comment:
The rule is to Flow type external interfaces, so you don't need to do this. You can if you want, however! It's up to you.